maskrcnn_benchmark/config/defaults.py

```diff
@@ -37,7 +38,7 @@
 # -----------------------------------------------------------------------------
 _C.INPUT = CN()
 # Size of the smallest side of the image during training
-_C.INPUT.MIN_SIZE_TRAIN = 800  # (800,)
+_C.INPUT.MIN_SIZE_TRAIN = (800,) # 800
```

Comment:
We are currently not using this I believe, and it conflicts with the changes in the Keypoints.
Maybe I'll just revert this part for now, or add it as a separate PR, are you ok with that?

Reply:
I am totally okay with that.